distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -363,13 +129,45 @@ public boolean matches(Object object) {
             "ddf.test.itests.platform.TestPlatform.startup.invalid");
 
     @BeforeExam
-    @SuppressWarnings("unchecked")
     public void beforeExam() throws Exception {
+        basePort = getBasePort();
         getAdminConfig().setLogLevels();
         getServiceManager().waitForAllBundles();
         console = new KarafConsole(bundleCtx);
     }
+    
+    public void resetInitialState() throws Exception {
+
+        FileUtils.deleteQuietly(getExportDirectory().toFile());
+        FileUtils.deleteQuietly(new File(TEST_FILE));
+
+        FileUtils.cleanDirectory(getPathToProcessedDirectory().toFile());
```

Comment:
I'll run this on windows and see how it goes. cleanDirectory() throws an exception if the directory doesn't exist. A similar thing happened with deleteDirectory() that caused some itests in testPlatform to fail when running on windows specifically.